include/crm/common/ordering.h

```diff
@@ -46,6 +46,9 @@ enum pe_ordering {
     //! 'first' becomes required if 'then' is required and for promoted instance
     pcmk_ord_promoted_then_implies_first    = (1U << 6),
 
+    //! 'first' is runnable only if 'then' is both runnable and migratable
```

Comment:
'first' also becomes required if 'then' is required

Reply:
thanks